test/date_time/dcm_date_time_test.dart

```diff
@@ -304,17 +262,17 @@ void main() {
       }
 
       final dt1 = DcmDateTime.fromMicroseconds(0);
-      print(dt1);
-      print(dt1.microseconds);
+      log..debug('dt1: $dt1')..debug(dt1.microseconds);
       expect(dt0.microseconds == 0, true);
 
-      print(dt0.inet);
-      print('microseconds: ${dt0.microseconds}');
-      print('year: ${dt0.year}');
-      print('month: ${dt0.month}');
-      print('day: ${dt0.day}');
-      print('hour: ${dt0.hour}');
-      print('minute: ${dt0.minute}');
+      log
+        ..debug(dt0.inet)
+        ..debug('microseconds: ${dt0.microseconds}')
```

Comment:
This set of debug statements could be changed to one with '''...''' across multiple lines